docs/specification/auth/credential.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Credential Authentication
```

Comment:
IMO this term is too ambiguous, since don't _all_ forms of "client giving details to the server" (including OAuth) involve some kind of credentials?

What about "token authentication" or "bearer authentication" or something like that? Neither is _quite_ expansive enough (e.g., to include use cases like providing a raw password), but that might be OK.

Reply:
I have no strong feelings about naming; my take is that OAuth is a fancy wrapper on top of this. Let's see if anyone is an advocate for a particular name and I can go rename the file + references to "credential".